.github/workflows/runtime-ci-cd.yaml

```
@@ -0,0 +1,124 @@
+name: runtime CI-CD
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "packages/runtime/**"
+      - ".github/workflows/runtime-ci-cd.yaml"
+  pull_request:
+    branches:
+      - master
+    paths:
+      - "packages/runtime/**"
+      - ".github/workflows/runtime-ci-cd.yaml"
+
+jobs:
+  ci:
+    name: Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      fail-fast: true
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: install node@14
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+
+      - name: yarn install
+        run: yarn install --immutable
+
+      - name: yarn lint
+        working-directory: packages/runtime
+        run: yarn lint
+
+      - name: yarn build
+        working-directory: packages/runtime
+        run: yarn build
+
+      - name: yarn test
+        working-directory: packages/runtime
+        run: yarn test
+        env:
+          HOME: ./
+
+  cd:
+    if: github.ref == 'refs/heads/master'
+    name: Publish to NPM
+    needs: ci
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    defaults:
+      run:
+        working-directory: packages/runtime
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+
+      - name: Configure git identity
+        run: |
+          # Configure git for commits
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+
+      - name: Install Node
+        uses: actions/setup-node@v2
+        with:
+          always-auth: true
+          node-version: 14
+
+      - name: Install dependencies
+        run: yarn install --immutable
+
+      - name: Build
+        working-directory: packages/runtime
+        run: yarn build
+
+      - id: publish
+        name: Publish to NPM
+        uses: JS-DevTools/npm-publish@v1
+        with:
+          package: packages/runtime/package.json
+          token: ${{ secrets.NPM_TOKEN }}
```

I think adding this should make the initial publish succeed:

```suggestion
          token: ${{ secrets.NPM_TOKEN }}
          checkVersion: false
          access: publish
```